decentraland/unity-explorer
# fix: Cannot see the changes made on other player's passport

## What does this PR change?
Fix #1553 
Fix #1634 

When we made any change from our Passport, the profile information was being published correctly but it was not being propagated through the world, so none of the other players were able to see the changes updated when they clicked on our avatar.

## How to test the changes?
1. Connect with two players, A and B.
2. With A, make changes on the profile and save.
3. Once it's done, with B click on the user and see the profile card (take into account that it could take several seconds in propagate the change).
4. Check that the info is correctly updated.

### Additional test (from Backpack)
Do the same test but equipping/unequipping wearables from the Backpack. We must check that none of the backpack's flow broke.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
